content-script.js

```diff
@@ -0,0 +1,482 @@
+// Content script for GitHub Draft Issues extension
+(function() {
+    'use strict';
+
+    // Draft storage manager
+    const DraftManager = {
+        STORAGE_KEY: 'github_draft_issues',
+        
+        // Get all drafts for current repository
+        async getDrafts() {
+            const result = await chrome.storage.local.get([this.STORAGE_KEY]);
+            const allDrafts = result[this.STORAGE_KEY] || {};
+            const repoKey = this.getCurrentRepoKey();
+            return allDrafts[repoKey] || {};
+        },
+        
+        // Save a draft
+        async saveDraft(id, draftData) {
+            const result = await chrome.storage.local.get([this.STORAGE_KEY]);
+            const allDrafts = result[this.STORAGE_KEY] || {};
+            const repoKey = this.getCurrentRepoKey();
+            
+            if (!allDrafts[repoKey]) {
+                allDrafts[repoKey] = {};
+            }
+            
+            allDrafts[repoKey][id] = {
+                ...draftData,
+                id: id,
+                repository: repoKey,
+                updatedAt: new Date().toISOString()
+            };
+            
+            await chrome.storage.local.set({ [this.STORAGE_KEY]: allDrafts });
+            return allDrafts[repoKey][id];
+        },
+        
+        // Delete a draft
+        async deleteDraft(id) {
+            const result = await chrome.storage.local.get([this.STORAGE_KEY]);
+            const allDrafts = result[this.STORAGE_KEY] || {};
+            const repoKey = this.getCurrentRepoKey();
+            
+            if (allDrafts[repoKey] && allDrafts[repoKey][id]) {
+                delete allDrafts[repoKey][id];
+                await chrome.storage.local.set({ [this.STORAGE_KEY]: allDrafts });
+                return true;
+            }
+            return false;
+        },
+        
+        // Get current repository key
+        getCurrentRepoKey() {
+            const pathParts = window.location.pathname.split('/').filter(part => part);
+            if (pathParts.length >= 2) {
+                return `${pathParts[0]}/${pathParts[1]}`;
+            }
+            return 'unknown';
+        },
+        
+        // Generate unique ID for draft
+        generateDraftId() {
+            return Date.now().toString(36) + Math.random().toString(36).substr(2);
+        }
+    };
+
+    // UI Manager for creating and managing draft interface
+    const UIManager = {
+        // Add Draft Issue button to GitHub UI
+        addDraftButton() {
```

Comment:
This doesn't appear to be working. When I go to `/issues/new`, I don't see a Draft button added. There should be one next to the Create button.